architecture/observability/README.md

```diff
@@ -0,0 +1,62 @@
+5. Enable the multi-cluster observability add-on. There are a couple of options to accomplish this:
+    - Via the OpenShift web console, select the installed operators and clicking on “Create instance” for MultiClusterObservability
+    - Create a Custom Resource (CR) and apply it
+6. Step #5 will generate a link for Grafana in the RHACM web console once all the necessary pods for Thanos, Grafana and Alertmanager are created, and all the managed clusters are connected to the RHACM Hub cluster and sending their metrics to the RHACM observability service
+7. Click on the Grafana link to launch the Grafana dashboard to validate that observability is enabled and metrics data is populated
+
+## Observability workflow
+
+1. The Metrics Collector in the managed clusters (Prod, Test, Obs) gets all the cluster platform level metrics from the local Prometheus in each cluster
+2. Observability Controller API collects the metric data from the Metrics Collector in each cluster and pushes them to Thanos for storing
+3. Thanos leverages Object Storage, in this case OpenShift Data Foundation residing in the Observability cluster (ocp-nerc-obs) to archive the collected metrics data
+4. Grafana dashboard on the Observability cluster (ocp-nerc-obs) will use Thanos Query to request the metrics data, and Thanos will serve up the metrics data from the object storage to the Grafana dashboard
+5. The Infrastructure will utilize it’s local Prometheus instance and push it’s platform metrics over to the ACM Hub using the local Metrics Collector
+6. The Observability cluster similar to the Infrastructure cluster will also utilize it’s local Prometheus instance and push it’s platform metrics over to the ACM Hub using the local Metrics Collector
+7. Configure ACM Alert Manager to resolve duplicate alerts through its dedupe process, group them and route the alerts to the appropriate receiving tool such as PagerDuty, DataDog, Splunk, email, etc.
+
+The architecture diagram below is a graphical overview of the observability solution.
+
+# Architecture Diagram
+
+![plot](./img/Observability-Architecture.png)
+
+# Technology
+
+The following technologies have been chosen for this solution:
+
+**Red Hat Advanced Cluster Management (RHACM):** This component helps overcome the challenges of managing multi-clusters environments and ensures appropriate enforcement of configurations and policies across the clusters for governance and compliance.
+
+**Red Hat Advanced Cluster Management Observability Service:** The observability service provides a centralized hub for metrics, alerting, and monitoring of platforms for a multi-cluster environment. In addition, the observability component also focuses on displaying cluster health metrics, which describes the control plane health, cluster optimization and resource utilization. gets deployed automatically to each cluster when the service is enabled.
```

Comment:
@jbasu01 My only feedback is this sentance could be capitalized "gets deployed automatically... " This documentation nicely describes all the components of metrics and alerts. Thanks very much for putting this together!

Reply:
Good catch. Actually, that sentence does fit there. I'll make the appropriate changes.